### PR TITLE
fixes bug 1091732 - changed JSONs from being fat strings

### PR DESCRIPTION
**Pull request by yellowsn0w**
Changed JSON big fat strings into dictionaries.

### CODE REVIEW

File: webapp-django/crashstats/crashstats/tests/test_models.py [hunk 1]
```diff
@@ -1105,7 +1064,7 @@ def mocked_put(url, **options):
             data = options['data']
             eq_(data['WaterWolf'], '18.0,19.0')
             eq_(data['NightTrain'], '1,2')
-            return Response("true")
+            return Response("True")
```
Comment (peterbe) [inline webapp-django/crashstats/crashstats/tests/test_models.py]: This should be possible to do with `return Response(True)`

**Comment by peterbe** [comment]
Hi! Thank you for getting started on this. 
I'm going to try to help you explain what's wrong with the isodate parsing of dates as string. 

Do you hang out on #breakpad? I'm peterbe there.

**Comment by peterbe** [comment]
Hi @yellowsn0w 
I'm sorry for being late responding to this one. The reason was I've been busy trying to finish this PR https://github.com/mozilla/socorro/pull/3223 first. That PR changes a lot of things and and it replaces a lot of places where JSON is typed as a big fat string, with completely different ways of doing things. 

The reason I've been rushing is because I'm preparing before my pending paternity leave. 

Would you mind waiting a couple of days and dealing with a rebase?

There are otherwise other good bugs to work on. I can help you find some good starter bugs.

**Comment by yellowsn0w** [comment]
Peter,

Its no problem at all! Don't worry about it!

Thank you for your help,
Denis Marchin

**Comment by peterbe** [comment]
Hi @yellowsn0w 
I'm sorry that the code had to change right under your feet. We helped you poorly, helping us. 

Please please try to find some other bug to work and I'll try to help you better next time.

**Comment by yellowsn0w** [comment]
Hey @peterbe, its no worries at all! Despite not being able to fully fix the issue, I still got nearly perfect on my assignment for the course I was doing this for! So thank you for your help 😊

**Pull request closed without merging**